modules/users/server/routes/admin.server.routes.js

```diff
@@ -4,22 +4,28 @@
  * Module dependencies
  */
 var adminPolicy = require('../policies/admin.server.policy'),
-  admin = require('../controllers/admin.server.controller');
+  admin = require('../controllers/admin.server.controller'),
+  passport = require('passport'),
+  express = require('express');
 
 module.exports = function (app) {
-  // User route registration first. Ref: #713
-  require('./users.server.routes.js')(app);
+  var router = express.Router();
 
-  // Users collection routes
-  app.route('/api/users')
+  //Set JWT Auth for all Admin Routes
+  router.all('*', passport.authenticate('jwt', { session: false }));
+
+  router.route('/')
     .get(adminPolicy.isAllowed, admin.list);
 
   // Single user routes
-  app.route('/api/users/:userId')
+  router.route('/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
     .delete(adminPolicy.isAllowed, admin.delete);
 
   // Finish by binding the user middleware
-  app.param('userId', admin.userByID);
+  router.param('userId', admin.userByID);
+
+  app.use('/api/admin/users', router);
```

Comment:
Because the admin and user would bind to /api/users.  If you look at the current code base it requires the users first to prevent it as a hack.  Otherwise the admin feature of /:userId matches /api/users/me and /api/users/password.  Regex prevents it.  Otherwise we have separate admin routes /api/admin/users.